Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 
 gem 'mongo_mapper', '0.12.0'
+gem 'mongo', '1.11.1'
+gem 'plucky', '0.5.2'
```

Comment:
This doesn't "lock it" anymore than Bundler ever locks it. It just prevents the gem developer from upgrading away from this version unintentionally.